### PR TITLE
fix: typos and 'private' functions

### PR DESCRIPTION
**Pull request by Mouwrice**

### CODE REVIEW

File: src/dice.py [hunk 1]
```diff
@@ -4,35 +4,35 @@
...
 
-def roll_dices(amount: int) -> [int]:
+def _roll_dice(amount: int) -> [int]:
     """
-    Simulates rolling a given amount of standard 6 sided dices
-    :param amount: The amount of dices to roll
-    :returns: A list of dice rolls
+    Simulates rolling a given amount of standard 6 sided dice
+    :param amount: The amount of dice to roll
+    :returns: A list of die rolls
```
Comment (NielsPraet) [inline src/dice.py:11]: You can keep this as `dice` ;)

File: src/dice.py [hunk 2]
```diff
@@ -4,35 +4,35 @@
...
+def _player_rolls_dice(player: Player, amount: int) -> ([int], str):
     """
-    Lets a player roll a given amount of standard 6 sided dices
-    :param player: The player that rolls the dice
-    :param amount: The amount of dices used
-    :returns: A tuple of (a list of dice rolls, a textual result)
+    Lets a player roll a given amount of standard 6 sided dice
+    :param player: The player that rolls the die
+    :param amount: The amount of dice used
+    :returns: A tuple of (a list of die rolls, a textual result)
```
Comment (NielsPraet) [inline src/dice.py:21]: Same here

File: src/dice.py [hunk 3]
```diff
@@ -4,35 +4,35 @@
...
-def players_roll_dice(players: [Player], verbose: bool) -> [int]:
+def _players_roll_die(players: [Player], verbose: bool) -> [int]:
     """
-    Rolls a standard 6 sided dice for every player
-    :param players: List of players to roll dices for
+    Rolls a standard 6 sided die for every player
+    :param players: List of players to roll dice for
     :param verbose: True if it should print the results to the console
-    :returns: A list of dice rolls
+    :returns: A list of die rolls
```
Comment (NielsPraet) [inline src/dice.py:33]: And here too

**Comment by lars-vc** [comment]
LGTM 👍

**Pull request merged**